src/main/java/io/beanmapper/strategy/MapToClassStrategy.java

```diff
@@ -12,15 +12,15 @@ public MapToClassStrategy(BeanMapper beanMapper, Configuration configuration) {
     }
 
     @Override
-    public Object map(Object source) {
+    public <S, T> T map(S source) {
         Class<?> targetClass = getConfiguration().getTargetClass();
 
-        if (getConfiguration().isConverterChoosable()) {
+        if (getConfiguration().isConverterChoosable() || source instanceof Record) {
             Class<?> valueClass = getConfiguration().getBeanUnproxy().unproxy(source.getClass());
             BeanConverter converter = getConverterOptional(valueClass, targetClass);
             if (converter != null) {
                 logger.debug("    {}->", converter.getClass().getSimpleName());
```

Comment:
Moet deze niet weg? Anders misschien wat extra info in de log wat het is. In MapToClassStrategy zat deze ook trouwens.

Reply:
Heb er wat extra info aan toegevoegd. Dat block is wel redelijk noodzakelijk.